subiquity/server/controllers/network.py

```diff
@@ -239,7 +238,10 @@ async def _call_client(self, client, conn, lock, meth_name, *args):
             if conn.closed:
                 log.debug('closed')
                 return
-            await getattr(client, meth_name).POST(*args)
+            try:
+                await getattr(client, meth_name).POST(*args)
+            except aiohttp.ClientError:
```

Comment:
Is it interesting to grab the text of the clienterror?  Otherwise LGTM.

Reply:
yeah probably actually